README.md

```diff
@@ -77,14 +76,25 @@ Example:
 
 input = '{"path": "/", "role": "admin"}';
 
-rego.load_policy(policy_wasm).then(policy => {
-    allowed = policy.eval_bool(input);
-    console.log("allowed = " + allowed);
+loadPolicy(policyWasm).then(policy => {
+    resultSet = policy.evaluate(input);
+    if (resultSet == null) {
+        console.error("evaluation error")
+    }
+    if (resultSet.length == 0) {
+        console.log("undefined")
+    }
+    console.log("allowed = " + allowed[0].result);
 }).catch( error => {
     console.error("Failed to load policy: " + error);
```

Comment:
Super random nit picker, but would worth passing `error` as a second argument instead of concating it, as that results in it getting fully printing out w/ stack trace.

Reply:
👍  Good idea, i'll update it.